cmake/AwsCFlags.cmake

```
@@ -250,6 +251,11 @@ function(aws_set_common_properties target)
         endif()
     endif()
 
+    if(AWS_ENABLE_TRACING)
+        target_link_libraries(${target} PRIVATE ittnotify)
+    else()
+        list(APPEND AWS_C_DEFINES_PRIVATE -DINTEL_NO_ITTNOTIFY_API) # Disable intel notify api if tracing is not enabled
```

utterly trivial: kind of a long comment to tack onto the end of a line
```suggestion
        # Disable intel notify api if tracing is not enabled
        list(APPEND AWS_C_DEFINES_PRIVATE -DINTEL_NO_ITTNOTIFY_API)
```